catalog/controller/extension/payment/razorpay.php

```diff
@@ -49,6 +50,30 @@ public function index()
             exit;
         }
 
+        try
+        {
+            $webhookUpdatedAt = $this->config->get('payment_razorpay_webhook_updated_at');
+
+            if($webhookUpdatedAt + 86400 < time())
+            {
+                $createWebhook = new CreateWebhook(
+                    $this->config->get('payment_razorpay_key_id'),
+                    $this->config->get('payment_razorpay_key_secret'),
+                    $this->config->get('payment_razorpay_webhook_secret'),
+                    HTTPS_SERVER . 'index.php?route=extension/payment/razorpay/webhook'
```

Comment:
@abdulwahidsharief , can't we define it as constant and use it.

Reply:
@ramth05 added as constant in both admin and catalog